ui/pages/confirmations/components/confirm/network-change-toast/network-change-toast.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import useCurrentConfirmation from '../../../hooks/useCurrentConfirmation';
+import NetworkChangeToastInner from './network-change-toast-inner';
+
+// The component has been broken into NetworkChangeToast and NetworkChangeToastInner
+// to suffice need of old and re-designed confirmation pages.
+// These can be merged once we get rid of old confirmation pages.
+const NetworkChangeToast = () => {
```

Comment:
Is it feasible to add a storybook for this?

Reply:
The component needs some actions, I am not sure how to mock those in storybook.

Reply:
https://github.com/MetaMask/metamask-extension/blob/develop/.storybook/main.js#L45

Though might be awkward so probably not needed for now.